ApplicationLibCode/FileInterface/RifFaultReactivationModelExporter.cpp

```diff
@@ -39,13 +39,13 @@ std::pair<bool, std::string> RifFaultReactivationModelExporter::exportToStream(
 {
     std::string applicationNameAndVersion = std::string( RI_APPLICATION_NAME ) + " " + std::string( STRPRODUCTVER );
 
-    using PartBorderSurface                                        = RigGriddedPart3d::BorderSurface;
+    using PartBorderSurface                                        = RimFaultReactivation::BorderSurface;
     std::vector<std::pair<PartBorderSurface, std::string>> borders = { { PartBorderSurface::UpperSurface, "TOP" },
                                                                        { PartBorderSurface::FaultSurface, "FAULT" },
                                                                        { PartBorderSurface::LowerSurface, "BASE" } };
 
     // The two parts are "mirrored", so face number 4 should of the two parts should face eachother.
```

Comment:
Typo in "so face number 4 should of the two parts should face eachother."

Reply:
Fixed.